### PR TITLE
state priorities in triaging and discussion process

### PR DESCRIPTION
**Pull request by fricklerhandwerk**
based on

- Nix team decisions
  https://discourse.nixos.org/t/2022-11-11-nix-team-meeting-minutes-7/23451#planning-discussion-1
  https://discourse.nixos.org/t/2022-12-02-nix-team-meeting-minutes-13/23731#discussion-3

- proposal to use labels more effectively

  https://discourse.nixos.org/t/improving-nix-developer-experience/21629

- documentation team decision to foster expressing interest using upvotes

  https://github.com/NixOS/nix/pull/7387

I added rationale for each of the priorities, which were not in the protocol but would hopefully help readers follow our reasoning.

### CODE REVIEW

**Review by Ericson2314 — APPROVED** [review]

**Comment by Ericson2314** [comment]
Great additions here!

File: maintainers/README.md [hunk 1]
```diff
@@ -36,17 +36,60 @@ Issues on the board progress through the following states:
...
+  - [documentation](https://github.com/NixOS/nix/labels/documentation)
+  - [UX](https://github.com/NixOS/nix/labels/UX)
+  - [error-messages](https://github.com/NixOS/nix/labels/error-messages)
+  - [new-cli](https://github.com/NixOS/nix/labels/new-cli)
+  - [flakes](https://github.com/NixOS/nix/labels/flakes)
+  - [developer-experience](https://github.com/NixOS/nix/labels/developer-experience)
+  - [macos](https://github.com/NixOS/nix/labels/macos)
+  - [windows](https://github.com/NixOS/nix/labels/windows) (new)
+  - [performance](https://github.com/NixOS/nix/labels/performance) (new)
+  - [ca-derivations](https://github.com/NixOS/nix/labels/ca-derivations)
```
Comment (edolstra) [inline maintainers/README.md:69]: Do we really need this list here? Team members can bring up whatever PRs/issues they want. They don't have to have any particular label.

Comment (fricklerhandwerk) [reply maintainers/README.md:69]: You're right. We don't require the list, it was meant as an overview. In practice we will probably mostly pick new things anyway, and label those in the process.

We could use the later though, if we decided to focus on a selection of topics, and add those to the categories to triage every week. Will remove it.

File: maintainers/README.md [hunk 2]
```diff
@@ -36,17 +36,45 @@ Issues on the board progress through the following states:
...
   If there is disagreement on the general idea behind an issue or pull request, it is moved to _To discuss_, otherwise to _In review_.
 
 - To discuss
 
-  Pull requests and issues that are important and controversial are discussed by the team during discussion meetings.
+  Pull requests and issues that are deemed important and controversial are discussed by the team during discussion meetings.
 
   This may be where the merit of the change itself or the implementation strategy is contested by a team member.
 
+  The order of items is determined as follows:
```
Comment (tomberek) [inline maintainers/README.md:64]:
```suggestion
  As a general guideline, the order of items is determined as follows:
```

**Comment by fricklerhandwerk** [comment]
Discussed in Nix team meeting 2023-01-06:

  - @thufschmitt: having old issues around should not be a problem
      - @fricklerhandwerk: disagree. they take a toll on attention, and signal how we groom the issue tracker
      - @roberth: old open issues also show up on search results, and are read by people, making the impression we don't care
  - @thufschmitt: the order makes it look like there is a firm rule, should reword it to be a general guideline
      - agreement
  - @fricklerhandwerk: will fix up according to suggestions

**Comment by fricklerhandwerk** [comment]
@thufschmitt updated according to suggestions.

**Review by thufschmitt — APPROVED** [review]
